rts/System/Platform/Win/WinVersion.cpp

```diff
@@ -11,7 +11,10 @@
 #endif
 
 #include <windows.h>
-#include <ntdef.h>
+//#include <ntdef.h>
```

Comment:
try flipping the include order, ntdef.h may need to go first

Reply:
I tried to do several things before proceeding with ugliness above. Order flip in particular fails with:
```
E0035	#error directive: "No Target Architecture"	engine-legacy	C:\Program Files (x86)\Windows Kits\10\Include\10.0.17134.0\shared\ntdef.h	179
E0337	linkage specification is incompatible with previous "Int64ShllMod32" (declared at line 1282 of "C:\Program Files (x86)\Windows Kits\10\Include\10.0.17134.0\shared\ntdef.h")	engine-legacy	C:\Program Files (x86)\Windows Kits\10\Include\10.0.17134.0\um\winnt.h	928
E0337	linkage specification is incompatible with previous "Int64ShraMod32" (declared at line 1298 of "C:\Program Files (x86)\Windows Kits\10\Include\10.0.17134.0\shared\ntdef.h")	engine-legacy	C:\Program Files (x86)\Windows Kits\10\Include\10.0.17134.0\um\winnt.h	935	
E0337	linkage specification is incompatible with previous "Int64ShrlMod32" (declared at line 1314 of "C:\Program Files (x86)\Windows Kits\10\Include\10.0.17134.0\shared\ntdef.h")	engine-legacy	C:\Program Files (x86)\Windows Kits\10\Include\10.0.17134.0\um\winnt.h	942	
```

Reply:
alright, no desire to dive into Windows internals atm